Extensions/Kayenta/Src/Tasks/createCanaryAnalysisRequestV0/task.json

```diff
@@ -0,0 +1,298 @@
+    "groupName": "scopes",
+    "properties": {
+        "EditableOptions": "True"
+      }
+  },
+  {
+    "name": "extendedScopeParams",
+    "type": "multiLine",      
+    "label": "Extended scope parameters",
+    "required": "false",
+    "helpMarkDown": "Will introduce additional scope parameters",
+    "groupName": "scopes",
+    "properties": {
+      "editorExtension": "ms.vss-services-azure.azure-servicebus-message-grid"
+     }
+  }
+],
+"dataSourceBindings": [{
+  "target": "canaryConfig",
+  "endpointId": "$(KayentaConnection)",
+  "dataSourceName": "canaryConfig",
+  "resultTemplate": "{ \"Value\" : \"{{id}}\", \"DisplayValue\" : \"{{name}}\" }",
+  "parameters": {
+    "configAccountName": "$(configAccountName)"
+}
+}
+],
+  "execution": {
+    "HttpRequestChain": {
+       "Execute": [
+        {
+          "RequestInputs": {
+            "EndpointId": "$(KayentaConnection)",
+            "EndpointUrl": "$(endpoint.url)standalone_canary_analysis/$(canaryConfig)?metricsAccountName=$(metricsAccountName)&configurationAccountName=$(configAccountName)&storageAccountName=$(storageAccountName)",
+            "Method": "POST",
+            "Body": "{\"analysisIntervalMins\":\"$(analysisIntervalMins)\",\"beginAfterMins\":\"$(release.deployment.gate.stabilizationTime)\",\"lifetimeDurationMins\":\"$(lifetimeDurationMins)\",\"lookbackMins\":\"$(lookbackMins)\",\"scopes\":[{\"controlLocation\":\"$(controlLocation)\",\"controlScope\":\"$(controlScope)\",{{#if endTimeIso}}\"endTimeIso\":\"$(endTimeIso)\",{{/if}}\"experimentLocation\":\"$(experimentLocation)\",\"experimentScope\":\"$(experimentScope)\",{{#if extendedScopeParams}}\"extendedScopeParams\":$(extendedScopeParams),{{/if}}\"scopeName\":\"$(scopeName)\",{{#if startTimeIso}}\"startTimeIso\":\"$(startTimeIso)\",{{/if}}\"step\":\"$(step)\"}],{{#if siteLocal}}\"siteLocal\":$(siteLocal),{{/if}}\"thresholds\":{\"marginal\":\"$(marginalThreshold)\",\"pass\":\"$(passThreshold)\"}}",
+            "Headers": "{\"Content-Type\":\"application/json\", \"Accept\":\"application/json\"}",
+            "WaitForCompletion": "false",
+            "Expression": "eq(isNullOrEmpty(jsonpath('$.canaryAnalysisExecutionId')[0]),false)"
+          },
+          "ExecutionOptions": {
+            "OutputVariables": "{\"canaryAnalysisExecutionId\":\"jsonpath('$.canaryAnalysisExecutionId')[0]\"}",
+            "SkipSectionExpression": "eq(isNullOrEmpty(variables['canaryAnalysisExecutionId']),false)"
+          }
+        },
+        {
+          "RequestInputs": {
+            "EndpointId": "$(KayentaConnection)",
+            "EndpointUrl": "$(endpoint.url)standalone_canary_analysis/$(canaryAnalysisExecutionId)",
+            "Method": "GET",
+            "WaitForCompletion": "false",
+            "Expression": "eq(eq(jsonpath('$.canaryAnalysisExecutionResult.canaryExecutionResults[0].result.judgeResult.score.classification')[0],'Pass'),true)"
+          }
```

Comment:
why 2 levels of nesting in Success Expression ? Simplify the expression to keep only 1 equals check.